src/util/json.h

```diff
@@ -28,6 +31,24 @@ Convenience header for Nlohmann::Json that sets the default options. Also
 #include "util/Exception.h"
 #include "util/SourceLocation.h"
 
+/*
+@brief Read the specified json file and build a json object out of it.
+
+@param jsonFileName Name of the file, or path to the file. Must describe a
+`.json` file.
+*/
+inline nlohmann::json fileToJson(std::string_view jsonFileName) {
+  // Check, if the filename/-path ends with ".json". Checking, if it's a valid
+  // file, is done by `ad_utility::makeIfstream`.
+  if (!jsonFileName.ends_with(".json")) {
+    throw std::runtime_error(absl::StrCat("The given filename/-path '",
+                                          jsonFileName,
+                                          "' doesn't describe a json file."));
+  }
+
+  return nlohmann::json::parse(ad_utility::makeIfstream(jsonFileName));
```

Comment:
I think you should catch the errors and throw something like
`The contents of the file <filename>.json could not be parsed as JSON. The error was: ` and then the error from `nlohmann`. This would make it much more user-friendly (ill-formed-JSON files are not really untypical).